include/models/relationship.hpp

```diff
@@ -0,0 +1,35 @@
+#ifndef _relationship_hpp_
+#define _relationship_hpp_
+
+#include <models/entity.hpp>
+#include <models/user.hpp>
+#include <models/relationship_type.hpp>
+
+namespace disccord
+{
+    namespace models
+    {
+        class relationship : public model

```

Comment:
I'm not sure whether this should inherit from entity, I need to check how relationships work